starter-code/js/index.js

```diff
@@ -2,16 +2,31 @@ function deleteItem(e){
 
 }
 
+
 function getPriceByProduct(itemNode){
 
+    var price = document.querySelector(itemNode);
+    return price;
+
+}
+
+function getQuantitybyProduct(itemNode){
+
+  var qty = document.querySelector(itemNode).innerHTML;
+  return qty;
+
 }
 
 function updatePriceByProduct(productPrice, index){
 
+  
+
 }
 
 function getTotalPrice() {
 
+  return getPriceByProduct(document.getElementsByClassName('.unit-cost')) * getQuantitybyProduct(document.getElementsByClassName('.quantity'));
```

Comment:
This line is too long and unreadable. Don't be afraid to break it down in to small steps. Readability is key!